.github/dependabot.yml

```diff
@@ -25,11 +25,15 @@ updates:
       prefix: chore(deps)
       prefix-development: chore(deps-dev)
     ignore:
+      # chalk is now an ESM package and can't be used here without a rewrite.
```

Comment:
Do we have a backlog item recorded anywhere to upgrade key projects so we can use ESM packages? It's going to be pretty painful if we get stuck on old versions of all our dependencies.

Reply:
Agreed. We don't have any backlog items for this and we haven't really talked about it at all. It's going to be really painful to upgrade everything because it's going to require us to go ESM across **everything** at the same time.

Reply:
I'll file a ticket about this and make sure it's on Dom's radar -- sounds like painful tech debt but something we'll need to deal with sooner or later.